controlpanel/api/cluster.py

```diff
@@ -87,6 +86,17 @@ def is_ignored_exception(e):
         return True
 
 
+def ignore_unwanted_exception(fn):
+    def wraps(*args, **kwargs):
+        try:
+            return fn(*args, **kwargs)
+
+        except Exception as e:
+            if not is_ignored_exception(e):
+                raise e
+    return wraps
```

Comment:
Should I leave this as is? If you are going to stop ignoring exceptions then if I revert it back to having a try except in each function then it will still have to be removed

Reply:
Yeah, ignore my comment